docs/book/.spellcheck.yml

```diff
@@ -0,0 +1,28 @@
+matrix:
+  - name: SPCheck
+    aspell:
+      lang: en
+    dictionary:
+      encoding: utf-8
+      wordlists:
+        - docs/book/spell-check-custom-words.txt
+    pipeline:
+      - pyspelling.filters.context:
+          context_visible_first: true
+          escapes: \\[\\`~]
+          delimiters:
+          # Ignore all code blocks
+          - open: '(?s)^(?P<open> *`{3,}\s*(\w+\s*,?\s*)+.*?)$'
+            close: '^( *`{3,})$'
```

Comment:
If this is the same in all repos, could we make these the default values so we don't need to provide them in every repo's CI config?

Reply:
Same as above, we can, but there might be cases in which you may want to modify this.

Reply:
Yes, I was thinking this could be the default value and it could be overridden at the repo level. This isn't blocking though